sources/settings-defaults/aws-dev/Cargo.toml

```diff
@@ -10,6 +10,4 @@ build = "../build-defaults.rs"
 path = "../defaults-toml.rs"
 
 [build-dependencies.bottlerocket-defaults-helper]
-git = "https://github.com/bottlerocket-os/bottlerocket-settings-sdk"
-tag = "bottlerocket-defaults-helper-v0.1.0"
-version = "0.1.0"
+workspace = true
```

Comment:
nit: these don't follow the format you've changed the rest to.

Reply:
whoops, fixed